Ports/lxt/patches/ReadMe.md

```diff
@@ -0,0 +1,11 @@
+# Patches for lxt on SerenityOS
+
+## `0001-Add-support-for-SerenityOS.patch`
+
+This patch adds support for SerenityOS.
+Specifially, it does the following things:
```

Comment:
This seems like it's missing a newline in between (commit titles and commit messages usually do that, even if it's not strictly necessary).

Reply:
resolved locally, push coming soon